octobot_evaluators/api/evaluators.py

```diff
@@ -47,16 +47,29 @@
                                time_frame=time_frame,
                                relevant_evaluators=relevant_evaluators)
         for evaluator_class in create_advanced_types_list(evaluator_parent_class, config)
-        for symbol in __get_symbols_to_create(evaluator_class, symbols)
-        for time_frame in __get_time_frames_to_create(evaluator_class, time_frames)
+        for symbol in _get_symbols_to_create(evaluator_class, symbols, config)
+        for time_frame in _get_time_frames_to_create(evaluator_class, time_frames)
     ]
 
 
-def __get_symbols_to_create(evaluator_class, symbols):  # TODO replace with python 3.8 by :=
-    return symbols if symbols and not evaluator_class.get_is_symbol_widlcard() else [None]
+def get_evaluator_classes_with_type(evaluator_type, config) -> list:
+    return create_advanced_types_list(EvaluatorClassTypes[evaluator_type], config)
 
 
-def __get_time_frames_to_create(evaluator_class, time_frames):  # TODO replace with python 3.8 by :=
+def _get_symbols_to_create(evaluator_class, symbols, config):
+    if evaluator_class.get_is_symbol_widlcard():
+        return [None]
+    if symbols:
+        return symbols
+    if config:
+        return _get_crypto_currencies(config)
```

Comment:
@Herklos social evaluators are not depending on a specific pair but on an identifier (Bitcoin, Ethereum, etc) that can be found in config[CONFIG_CRYPTO_CURRENCIES].keys() => I added it there because I don't really know where else to put it, I would like to have your input on it.

Reply:
You mean to have `_get_crypto_currencies` in OctoBot-Evaluators API. I think it's the good place until we need it internally.